src/app/components_ngrx/work-item-detail/work-item-detail.component.ts

```diff
@@ -78,6 +78,8 @@ export class WorkItemDetailComponent implements OnInit, OnDestroy, AfterViewChec
       this.listenToEsc = true;
     }
   }
+  @Input() workItemTypes: WorkItemTypeUI[] = [];
+  selectedType: WorkItemTypeUI = null;
```

Comment:
What is the use of this variable? You can directly use the relationship from work item, can't you?

Reply:
I am using this variable so that as soon as the user clicks on a witype it gets updated in the UI, here
`<span class="dropdown-text">
      {{ selectedType ? selectedType.name : '' }}         
</span>
`

Reply:
You can still do that I believe.